charts/telepresence/templates/trafficManagerRbac/cluster-scope.yaml

```diff
@@ -56,6 +56,7 @@ rules:
   - delete
   resourceNames:
   - telepresence-agents
+  - telepresence-excluded-variables
```

Comment:
Using a configmap here is quite powerful. Could be be made more generic? Perhaps name it `telepresence-intercept-env` and then have an `excluded` top-level key under `data`?